pkg/doc/verifiable/jwt_unsecured_test.go

```diff
@@ -62,28 +63,36 @@ func TestUnmarshalUnsecuredJWT(t *testing.T) {
 
 	t.Run("rejects serialized JWT headers not in base64 format", func(t *testing.T) {
 		invalidJWT := fmt.Sprintf("%s.eyJwcm9kdWN0SWRzIjpbMSwyXSwic3ViIjoidXNlcjEyMyJ9.", notBase64)
-		_, _, err := unmarshalUnsecuredJWT([]byte(invalidJWT))
+		h, b, err := unmarshalUnsecuredJWT([]byte(invalidJWT))
```

Comment:
What do "h" and "b" stand for?

Reply:
:) `headers` and `body`. I'll rename it.